packages/twenty-front/src/modules/billing/components/internal/PlanTag.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Tag } from 'twenty-ui/components';
+import { t } from '@lingui/core/macro';
+import { BillingPlanKey } from '~/generated-metadata/graphql';
+
+export type PlanTagProps = {
+  plan: BillingPlanKey;
+  isTrialPeriod?: boolean;
+};
+
+export const PlanTag = ({ plan, isTrialPeriod = false }: PlanTagProps) => {
```

Comment:
this component is returning two tags and called PlanTag, this is a bit confusing